src/question.js

```diff
@@ -20,12 +20,21 @@ router.post('/check/:qno(\\d+)?', middleware.isAuthenticated, (req, res) => {
   var { qno } = req.params;
   const { answer } = req.body;
   const { lastQuestionAllowed } = req.user;
+  const {score} = req.user;
   if (!qno) qno = lastQuestionAllowed;
   if (qno > lastQuestionAllowed) {res.sendStatus(403); return false;}
   models.Question.findOne({where : { qno }})
     .then(question => {
       if (question) {
-        if (question.answer == answer && qno ==lastQuestionAllowed) req.user.update({lastQuestionAllowed: lastQuestionAllowed + 1});
+        if (question.answer == answer && qno ==lastQuestionAllowed)
+        {
+	  req.user.update({ score: score + 10});
```

Comment:
Wouldn't be better if a variable scoreIncrementor be added to score to score instead of 10 so that the scoring system be changed later easily? @meghprkh your thoughts? Also similarly a scoreDecrementor for incorrect responses whose value is 0 by default.